presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -296,4 +370,29 @@ private long calculateTotalRows(int scaleBase, double scaleFactor)
         }
         return (long) totalRows;
     }
+
+    public static Type getNumericType(TypeSignature numericTypeSignature)
+    {
+        if (numericTypeSignature.getBase().equals(DECIMAL)) {
```

Comment:
I was not part of this discussion. But it makes sense to me. 
It seems to me that if we want that approach we can safely drop this commit from PR and change it to desired shape in separate - single commit - PR.

I will try to run the build with this commit dropped and see if it passes.

@sopel39 what do you think.